polaris-service/src/main/java/org/apache/polaris/service/catalog/BasePolarisCatalog.java

```diff
@@ -1143,12 +1143,19 @@ private class BasePolarisCatalogViewBuilder extends BaseMetastoreViewCatalog.Bas
 
     public BasePolarisCatalogViewBuilder(TableIdentifier identifier) {
       super(identifier);
+      withProperties(viewDefaultProperties());
     }
 
     @Override
     public ViewBuilder withLocation(String newLocation) {
       return super.withLocation(transformTableLikeLocation(newLocation));
     }
+
+    private Map<String, String> viewDefaultProperties() {
+      Map<String, String> viewDefaultProperties = PropertyUtil.propertiesWithPrefix(BasePolarisCatalog.this.properties(), "table-default.");
+      LOGGER.info("View properties set at catalog level through catalog properties: {}", viewDefaultProperties);
```

Comment:
Let's make this a `DEBUG` log

Reply:
Thanks @sfc-gh-emaynard for reviewing. 
Do you think it might log any sensitive message? Since catalog properties could have some kind of credentials.

Reply:
That's a fair point. I'm also okay with removing the log. I just don't think it should be at the `INFO` level